SyncIntegrationTests/test_tabs_desktop.js

```diff
@@ -19,7 +19,13 @@ var tabs1 = [
 
 // sync and verify tabs
 Phase("phase1", [
-  [Sync],
   [Tabs.add, tabs1],
   [Sync]
 ]);
+
+Phase("phase2", [
```

Comment:
This workaround should work even after the fix in TPS is in.
https://bugzilla.mozilla.org/show_bug.cgi?id=1818349#c7